source/Sashimi/Sashimi.csproj

```diff
@@ -23,9 +23,9 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Octopus.Configuration" Version="4.1.0" />
-    <PackageReference Include="Sashimi.Azure.Accounts" Version="14.1.3" />
-    <PackageReference Include="Sashimi.Azure.Common" Version="14.1.3" />
+    <PackageReference Include="Sashimi.Azure.Accounts" Version="14.2.0" />
+    <PackageReference Include="Sashimi.Azure.Common" Version="14.2.0" />
     <PackageReference Include="Sashimi.AzureScripting" Version="14.1.0" />
```

Comment:
`Sashimi.AzureScripting` can be bumped to `14.2.0` too.

Reply:
Done, thanks